backend/adapter_processor/serializers.py

```diff
@@ -94,6 +79,25 @@ def to_representation(self, instance: AdapterInstance) -> dict[str, str]:
         return rep
 
 
+class AdapterInfoSerializer(BaseAdapterSerializer):
```

Comment:
NIT: Since we return this `context_window_size` with the `AdapterInstanceSerializer` why do we need this `AdapterInfoSerializer`?

Reply:
Answered [here](https://github.com/Zipstack/unstract/pull/257#issuecomment-2058219036)